saleor/graphql/product/mutations/attributes.py

```diff
@@ -300,7 +300,7 @@ class Arguments:
 
     class Meta:
         description = "Assign attributes to a given product type."
-        error_type_class = ProductError
+        error_type_class = ProductAttributeError
         error_type_field = "product_errors"
```

Comment:
If `error_type_class` is changed we probably should also change `error_type_field`.

Reply:
I thought about this, and I'm not sure if in this case we should change it